integration/main_test.go

```diff
@@ -218,6 +218,20 @@ func WithPodLabels(kvs map[string]string) PodSandboxOpts {
 	}
 }
 
+// WithSecurityContext set container privileged.
+func WithPodSecurityContext(privileged bool) PodSandboxOpts {
+	return func(p *runtime.PodSandboxConfig) {
+		if p.Linux == nil {
+			p.Linux = &runtime.LinuxPodSandboxConfig{}
+		}
+		if p.Linux.SecurityContext == nil {
+			p.Linux.SecurityContext = &runtime.LinuxSandboxSecurityContext{
+				Privileged: privileged,
+			}
+		}
```

Comment:
nit: this means if `p.Linux.SecurityContext` is empty not nil, `privileged` will be skipped. maybe change to

```golang
if p.Linux.SecurityContext == nil {
  p.Linux.SecurityContext = &runtime.LinuxSandboxSecurityContext{}
}
p.Linux.SecurityContext.Privileged = privileged
```

same for below